packages/integrations/cloudflare/src/entrypoints/server.ts

```diff
@@ -97,7 +97,13 @@ export function createExports(manifest: SSRManifest) {
 			},
 		};
 
-		const response = await app.render(request, { routeData, locals });
+		const response = await app.render(request, {
+			routeData,
+			locals,
+			prerenderedErrorPageFetch: async (url) => {
+				return env.ASSETS.fetch(url.replace(/\.html$/, ''));
```

Comment:
This seems wrong at the first place, but it is.. It doesn't really make sense to force the user to add a service binding to the same worker. This also makes sure that it get's served for free if it is pre-rendered, as well as having a fallback to the server side runtime.

Reply:
Ah, nice. I didn't realise that would work